home_flask_1.py

```diff
@@ -14,16 +14,22 @@ def index():
 #passwoerd
 @app.route('/password')
 def generate_password():
-    while True:
-        # создаем рандомній список 10-20 символов
-        password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
-        # конвертируем список в строку
-        pas_check = ''.join(password)
-        # проверяым регулярным выражением на соответствие пароля ТЗ
-        if re.match(r'(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[!"#$%&(){}\'*+,-./\\:;<>=?@\[\]^_`|~])', pas_check):
-            break
+    password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
```

Comment:
why do you use randrange here? And why do you have this numbers 33 and 127?

Reply:
"And why do you have this numbers 33 and 127?....." This ASCII code password symbols

Reply:
This line generates from 10 to 20 codes ASCII, and translates them into a list of characters... Still, comments are sometimes help..)))

> why do you use randrange here? And why do you have this numbers 33 and 127?